### PR TITLE
2to3: add pt_BR translation

### PR DESCRIPTION
**Pull request by pedrokpp**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.pt_BR/common/2to3.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# 2to3
+
+> Conversão automática de código Python 2 para Python 3.
+> Mais informação: <https://docs.python.org/3/library/2to3.html>.
+
+- Mostrar as alterações que seriam feitas sem faze-las de fato (simulação):
+
+`2to3 {{diretorio/do/arquivo.py}}`
```
Comment (marchersimon) [inline pages.pt_BR/common/2to3.md:8]:
```suggestion
`2to3 {{caminho/para/arquivo.py}}`
```
I think `caminho/para/arquivo` is what we use in other pages.

**Review by Waples — APPROVED** [review]

File: pages.pt_BR/common/2to3.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`2to3 --write {{diretorio/do/arquivo.py}} --nofix={{has_key}} --nofix={{isinstance}}`
+
+- Mostrar a lista de todas os recursos disponíveis que podem ser convertidas de Python 2 para Python 3:
+
+`2to3 --list-fixes`
+
+- Converter todos os arquivos feitos em Python 2 em um diretório para Python 3:
+
+`2to3 --output-dir={{diretorio/que_contem/arquivos_python3}} --write-unchanged-files --nobackups {{diretorio/que_contem/arquivos_python2}}`
```
Comment (navarroaxel) [inline pages.pt_BR/common/2to3.md:28]: I don't like the main page for this example:
```
--output-dir=path/to/python3_directory
```
sound like the installation directory for python3. :thinking:

Comment (pedrokpp) [reply pages.pt_BR/common/2to3.md:28]: Hey! I also thought a bit like how you thought, that's why I changed it in the translation.

`diretorio/que_contem/arquivos_python3` means something like `path/that_contains/python3_files`

File: pages.pt_BR/common/2to3.md (hunk 1 above)
Comment (pedrokpp) [reply pages.pt_BR/common/2to3.md:8]: Sorry! I'm new a new contributor in general, I've never contributed in other repos 😄  lol

I'll try my best and see other's example to keep the patterns of all translations.

Comment (navarroaxel) [reply pages.pt_BR/common/2to3.md]: don't worry @pedrokpp, we have a few pages with `caminho/do` maybe we should standardize that. Most pages are with `caminho/paga`

Comment (bl-ue) [reply pages.pt_BR/common/2to3.md]: Either of you know about #5875?

File: pages.pt_BR/common/2to3.md (hunk 2 above)
Comment (bl-ue) [reply pages.pt_BR/common/2to3.md:28]: @navarroaxel that's a good point.  @sbrl do you think `path/that_contains/python3_files` is fine?

Comment (navarroaxel) [reply pages.pt_BR/common/2to3.md:28]: I checked this in Ubuntu focal (Docker) and `more v2.34` needs the `+10` before the file.

Comment (sbrl) [reply pages.pt_BR/common/2to3.md:28]: What about `python3_output`?

Comment (marchersimon) [reply pages.pt_BR/common/2to3.md:28]: Ping @pedrokpp / @bl-ue

File: pages.pt_BR/common/2to3.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+`2to3 --write {{diretorio/do/arquivo.py}}`
+
+- Converter recurso específico de Python 2 para Python 3:
+
+`2to3 --write {{diretorio/do/arquivo.py}} --fix={{raw_input}} --fix={{print}}`
+
+- Converter todos os recursos de Python 2 para Python 3, exceto as que específicadas:
+
+`2to3 --write {{diretorio/do/arquivo.py}} --nofix={{has_key}} --nofix={{isinstance}}`
```
Comment (navarroaxel) [inline pages.pt_BR/common/2to3.md:20]:
```suggestion
`2to3 --write {{caminho/para/arquivo.py}} --nofix={{has_key}} --nofix={{isinstance}}`
```

File: pages.pt_BR/common/2to3.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+`2to3 {{caminho/para/arquivo.py}}`
+
+- Converter um arquivo feito em Python 2 para Python 3:
+
+`2to3 --write {{diretorio/do/arquivo.py}}`
+
+- Converter recurso específico de Python 2 para Python 3:
+
+`2to3 --write {{diretorio/do/arquivo.py}} --fix={{raw_input}} --fix={{print}}`
```
Comment (navarroaxel) [inline pages.pt_BR/common/2to3.md:16]:
```suggestion
`2to3 --write {{caminho/para/arquivo.py}} --fix={{raw_input}} --fix={{print}}`
```

File: pages.pt_BR/common/2to3.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+> Conversão automática de código Python 2 para Python 3.
+> Mais informação: <https://docs.python.org/3/library/2to3.html>.
+
+- Mostrar as alterações que seriam feitas sem faze-las de fato (simulação):
+
+`2to3 {{caminho/para/arquivo.py}}`
+
+- Converter um arquivo feito em Python 2 para Python 3:
+
+`2to3 --write {{diretorio/do/arquivo.py}}`
```
Comment (navarroaxel) [inline pages.pt_BR/common/2to3.md:12]:
```suggestion
`2to3 --write {{caminho/para/arquivo.py}}`
```

File: pages.pt_BR/common/2to3.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+`2to3 --list-fixes`
+
+- Converter todos os arquivos feitos em Python 2 em um diretório para Python 3:
+
+`2to3 --output-dir={{diretorio/que_contem/arquivos_python3}} --write-unchanged-files --nobackups {{diretorio/que_contem/arquivos_python2}}`
+
+- Executar 2to3 com múltiplas threads:
+
+`2to3 --processes={{4}} --output-dir={{diretorio/que_contem/arquivos_python3}} --write --nobackups --no-diff {{diretorio/que_contem/arquivos_python2}}`
```
Comment (navarroaxel) [inline pages.pt_BR/common/2to3.md:32]:
```suggestion
`2to3 --processes={{4}} --output-dir={{caminho/para/arquivos_python3}} --write --nobackups --no-diff {{caminho/para/arquivos_python2}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
@pedrokpp could you apply my suggestions so we can merge?

**Comment by marchersimon** [comment]
@navarroaxel I think we can commit the changes ourselves by now.

**Review by marchersimon — APPROVED** [review]

File: pages.pt_BR/common/2to3.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+# 2to3
+
+> Conversão automática de código Python 2 para Python 3.
+> Mais informação: <https://docs.python.org/3/library/2to3.html>.
```
Comment (bl-ue) [inline pages.pt_BR/common/2to3.md:4]: We normally use `Mais informações` for this label.  Any reason for doing it differently?
```suggestion
> Mais informações: <https://docs.python.org/3/library/2to3.html>.
```

Comment (navarroaxel) [reply pages.pt_BR/common/2to3.md:4]: Now I read it. We use a plural like informations. 🤔

**Review by marchersimon — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]

**Review by CleanMachine1 — APPROVED** [review]
Can't speak Portuguese however looks alright

**Pull request merged**